### PR TITLE
Admin documentation

### PR DESCRIPTION
**Pull request by paulowenbrown**
## Description of changes
Added documentation for the new admin panel and updated outdated docs for pages within the administrative section.
Additionally removed Statistics page and associated tests, and fixed a url issue with the Remote Api page.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] ~CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
* [ ] ~Tests added (or description of how to test) for any new features.~
* [x] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/user/administrator/index.md [hunk 1]
```diff
@@ -23,6 +42,14 @@ This section of the guide describes how to create, modify, and remove local user
 Creating a New User Account
 ---------------------------
 
+First navigate to the users page via the admin panel side menu:
+
+![Users side menu link.](images/users-side-menu.png)
+
+Then press the "Add User" button:
```
Comment (deepsidhu85) [inline doc/user/administrator/index.md:49]: Change `press` to `click`

File: doc/_includes/tutorials/common/managing-user-groups.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+The user group role is saved as soon as you make a selection -- you **do not** need to click a "Save" button.
+
+### Removing a user from a user group
+
+You may want to completely remove all permissions for a user to access data in any project that the group is assigned to. To remove those permissions, you must remove the user account from the user group.
+
+Start by [viewing the user group details](#viewing-user-group-details).
+
+To remove a user group member, click on the button on the right-hand side of the table:
```
Comment (deepsidhu85) [inline doc/_includes/tutorials/common/managing-user-groups.md:55]: Change to `click on the "X" button on the right-hand side of the table`

File: doc/user/tutorials/sharing/index.md [hunk 3]
```diff
@@ -25,6 +25,12 @@ Creating a User Account
 
 If the person you would like to share data with within IRIDA does not have a user account, you'll need to make an account for them.
 
+You may create a new user account in IRIDA if you have the **Manager** or **Administrator** system role.
+
+After logging in to IRIDA, click on the **Users** menu, and select "Create User":
```
Comment (deepsidhu85) [inline doc/user/tutorials/sharing/index.md:30]: In `You may create a new user account in IRIDA if you have the **Manager** or **Administrator** system role` make the Administrator a link to the section in the Administrator guide as there are separate instructions for Admins


Change to:

After logging into IRIDA, click the gear icon in the top right hand side of the navbar and click **Create User**.

With this update the screenshot as well to also have the gear icon outlined

File: doc/user/user/user-groups/index.md [hunk 4]
```diff
@@ -32,75 +32,6 @@ The projects list provides a high-level overview of project details, including:
 * The date that the group was last modified, and
 * (if you are permitted) group deletion buttons.
 
-Searching the Groups Table
---------------------------
+{% include tutorials/common/managing-user-groups.md %}
 
-You can filter the user groups in IRIDA using the text-field at the top of the groups table:
```
Comment (deepsidhu85) [inline doc/user/user/user-groups/index.md:38]: Looks like the filter instructions weren't copied over to managing-user-groups.md. Is there a reason for that?

Comment (paulowenbrown) [reply doc/user/user/user-groups/index.md:38]: I removed them due to the instructions just detailing that there is a search bar. These instructions were not present for any of the other tables that all had search bars. Would you want me to add it back for user groups? Or add it for every table? Or just keep it removed?

**Pull request merged**